benchmark/text-generation-inference/llama3-70b-inf2.48xlarge/tgi-results-batchsize-1.csv

```diff
@@ -0,0 +1,11 @@
+model_id,concurrent requests,throughput (t/s),Time-to-first-token @ P50 (s),average latency (ms)
```

Comment:
I don't think batch_size 1 is relevant for a TGI server: do you really want to include it ?

Reply:
The goal here was to answer the question "how low can I get TTFT", and it shows the corresponding lack of changes in t/s (i.e. that is why you don't want a batch size of 1).  Less interesting and more educational.  I wrote this with a single customer in mind.  If it is not what you want for the benchmarking section we can drop it, your call.

Reply:
Understood. Let's keep it.